Source/Common/GLTFUnarchiver.swift

```diff
@@ -785,7 +790,8 @@ public class GLTFUnarchiver {
         }
         let glImage = images[index]
         
-        var image: Image?
+        // An Image or a URL
+        var image: Any?
```

Comment:
Same comment regarding `Any` usage as the above comments ([first comment](https://github.com/magicien/GLTFSceneKit/pull/34/files#diff-15843db1746890d478f30fbca4e35f5cR39), [second comment](https://github.com/magicien/GLTFSceneKit/pull/34/files#diff-15843db1746890d478f30fbca4e35f5cR779)).